### ISSUE TITLE
Naming

### ISSUE THREAD

**Issue by kegl** [paris-saclay-cds/ramp-workflow#20]
Consider renaming train_is to train_idxs and labels to label_names.

**Comment by aboucaud** [comment]
I guess you meant `train_is` to `train_ids` or `train_idx`

**Comment by glemaitre** [comment]
Since that there is couple of occurences with `idxs` in scikit-learn, I would agree with it :D

https://github.com/scikit-learn/scikit-learn/search?utf8=%E2%9C%93&q=idxs&type=

**Comment by jorisvandenbossche** [comment]
But most of the time sklearn uses either just `train, test = cv.splilt(..)` or `train_index, test_index = cv.split(..)`
